### PR TITLE
Reduces availability of Harpoon and Teleporter module.

### PR DESCRIPTION
**Pull request by Kurgis**
## About The Pull Request

It is odd how a device apparently so rare, and coming from a now extinct producer comes in such a great abundance. The Teleporter module never was once obtainable, and was designed for use by a Ninja antagonist on Bay who was meant to be peaceful. Not empty out the entire ship into space, as does every Traitor who discovers this effective function.

Moebius may no longer produce the NT Harpoon through the protolathe (which far as I'm aware was from an undocumented change); random suit module spawners will no longer spawn ninja gear, only rare random suit module spawners; the Expedition Overseer no longer has a second Bluespace Harpoon spawn within their Quarters, they must make use of the one within their office.

### CODE REVIEW

**Comment by SpadesSlick** [comment]
It will be more wise to lock Harpoon behind costly research.

**Comment by drexample** [comment]
I'm fine with BS harpoon being un-produceable. More reasons to add it as dungeon loot.

File: code/game/objects/random/rig.dm [hunk 1]
```diff
@@ -90,10 +89,6 @@
 	/obj/item/rig_module/fabricator/energy_net = 0.2,
 	/obj/item/rig_module/self_destruct = 1,
 
-	//Stealth
-	/obj/item/rig_module/stealth_field = 0.5,
-	/obj/item/rig_module/teleporter = 0.5,
```
Comment (ACCount12) [inline code/game/objects/random/rig.dm]: I think stealth field could stay. It's an awful power hog, making it useless for anything besides just "getting a drop" on someone - and I think that having a piece of rare loot enable that is fine.

Comment (Kurgis) [reply code/game/objects/random/rig.dm]: There's a second spawner called "rare rig module" which still spawns these.

Comment (Kurgis) [reply code/game/objects/random/rig.dm]: Will resolve.

**Comment by Kurgis** [comment]
I'm aware this PR has gone stale, I am still going to make the requested changes.

**Comment by drexample** [comment]
@Kurgis 
Are you planning on finishing the PR?

**Comment by drexample** [comment]
Winter clean up is here. This PR will be closed in 7 days, unless you plan to finish it.

**Pull request closed without merging**